kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/utils/DrawableUtils.kt

```diff
@@ -23,11 +24,30 @@ fun getBitmap(drawable: Drawable): Bitmap {
         getBitmapFromVectorDrawable(drawable)
 }
 
-fun compareDrawables(lhs: Drawable, rhs: Drawable) = lhs.constantState == rhs.constantState
+/**
+ * Compare drawables's [Drawable.ConstantState] and also [Bitmap]s from those drawables
+ * Only compares the [Drawable.ConstantState] can ended up in a possible false negative as stated here:
+ *
+ * https://stackoverflow.com/questions/9125229/comparing-two-drawables-in-android/31562099#31562099
+ *
+ * @param expected  Expected Drawable to be matched
+ * @param actual    Actual Drawable
+ *
+ * @return          True if they are identical, else otherwise.
```

Comment:
I think there's a little typo here on "else otherwise". You probably meant "false otherwise", correct me if I'm wrong

Reply:
Haha thanks for pointing that out, I'll fix asap